hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -4,8 +4,63 @@
 #
 # Meridian RDC Reset Scenario
 
+###############
+# Reset Group #
+###############
+
+# List of external reset group
+set ext_rst_lst {}
+# POR_N active low
+lappend ext_rst_lst [list { POR_N } {reset low}]
+
+foreach_in_collection j [get_pins top_earlgrey.rstmgr_aon_resets.*] {
+    lappend ext_rst_lst [list [get_attribute ${j} full_name] {reset low}]
+}
+
+set_reset_group $ext_rst_lst -name ext_rst
```

Comment:
does this imply they can all be asserted independently? if it's like "clock groups", @jeoongp or i can help to make this more intelligent

Reply:
It seems to be a little bit more complex than that. The example shows the reset group is more like relationship between the resets. For example, one reset signal could belongs to multiple reset groups.